sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3630,6 +3630,35 @@ object functions {
     window(timeColumn, windowDuration, windowDuration, "0 second")
   }
 
+  /**
+   * Generates session window given a timestamp specifying column.
+   *
+   * Session window is the one of dynamic windows, which means the length of window is vary
+   * according to the given inputs. The length of session window is defined as "the timestamp
+   * of latest input of the session + gap duration", so when the new inputs are bound to the
+   * current session window, the end time of session window can be expanded according to the new
+   * inputs.
+   *
+   * Windows can support microsecond precision. Windows in the order of months are not supported.
```

Comment:
> "Windows in the order of months are not supported."

I don't get what this sentence mean.

Reply:
We don't support the interval duration of window equal or more than a month. This is borrowed from `window` method doc, but I guess I need to change Windows to gapDuration to make clear.